api-controllers/ServiceFabrikApiController.js

```diff
@@ -11,6 +11,7 @@ const eventmesh = require('../data-access-layer/eventmesh');
 const lockManager = eventmesh.lockManager;
 const errors = require('../common/errors');
 const BackupService = require('../managers/backup-manager');
+const RestoreService = require('../managers/restore-manager');
```

Comment:
Why do we have reference to service from controller? I remember we kept a reference to `BackupService` for `getLastBackup`.
I believe this logic will be removed once we change our `getLastBackup` logic from getting `backupGuid` from `deployment` resource and then getting `backup` resource to `labelQuery` approach

Reply:
Its it to handle restores which are in progress while SF transitions from v1 restore to v2 restore